qulice-maven-plugin/src/main/java/com/qulice/maven/DefaultMavenEnvironment.java

```diff
@@ -238,10 +239,11 @@ public boolean exclude(final String check, final String name) {
 
     @Override
     public Collection<String> excludes(final String checker) {
-        return Collections2.transform(
+        final Collection<String> excludes = Collections2.transform(
```

Comment:
@vkuchyn please update your comment with the person you are addresing it to (the `@`) at the beginning